tests/test_config_consistency.py

```diff
@@ -303,3 +303,45 @@ def test_default_service_name(self):
         assert (
             spans[0]["service"] != "service_test"
         )  # in default scenario, DD_SERVICE is set to "weblog" in the dockerfile; this is a temp fix to test that it is not the value we manually set in the specific scenario
+
+
+@scenarios.tracing_config_nondefault
+@features.tracing_configuration_consistency
+class Test_Config_IntegrationEnabled_False:
```

Comment:
Just a small cherry on the cake, if you could add `@rfc` with the RFC link to decorate the class. It's optionnal, and does not change anything for the execution, but it helps a lots other users

Reply:
Addressed in latest commit.